persist/fs/fs_mock.go

```diff
@@ -27,8 +27,8 @@ import (
 	gomock "github.com/golang/mock/gomock"
 	ts "github.com/m3db/m3db/ts"
 	checked "github.com/m3db/m3x/checked"
-	time "github.com/m3db/m3x/time"
-	time0 "time"
+	time0 "github.com/m3db/m3x/time"
```

Comment:
could you drop unchanged files.

Reply:
Isn't this changed with the addition to `fs`?

Reply:
The only changes I see in this file are for the import alias rename, no signature changes beyond that yea?

Reply:
Fair point, I just assumed it changed bc I edited the `fs` file itself. I'll remove it.